### ISSUE TITLE
Window Topbar --> Thumbnail strip toggle on / off

### ISSUE THREAD

**Issue by camillevilla** [ProjectMirador/mirador#1758]
Depends on #1750

<img width="250" alt="views_and_thumbnails_v_2" src="https://user-images.githubusercontent.com/5402927/52028671-aa3fab00-24c4-11e9-87dd-80b4ea8e8e96.png">

<img width="635" alt="views_and_thumbnails_v_2-2" src="https://user-images.githubusercontent.com/5402927/52028706-d2c7a500-24c4-11e9-9f14-9b56f18456df.png">

<img width="239" alt="views_and_thumbnails_v_2-3" src="https://user-images.githubusercontent.com/5402927/52028669-aa3fab00-24c4-11e9-96a5-52238c645ed8.png">

### Resources
- [Interactive designs](https://github.com/ProjectMirador/mirador-design/wiki/Mirador-3-Interactive-Designs) --> View menu & thumbnail strip behaviour (Jan 29, 2019)
- Initial thumbnail nav implementation: #1661 

### Acceptance criteria
- [x] Clicking on the "Bottom" icon opens the thumbnail strip in the bottom companion window
- [x] Closing the "Off" icon closes the bottom companion window
- [ ] Opening / closing thumbnail stripshould not change the size of the window viewer area

### Out of scope
- right companion window behavior (select from menu or use drag button)

**Comment by cbeer** [comment]
I'm going to split:

> Opening / closing thumbnail stripshould not change the size of the window viewer area

out into its own ticket, 👌 ? I'm hoping that'll get address holistically with the companion window work.

**Comment by camillevilla** [comment]
I'm fine with closing this ticket now. This whole window resizing question affects several features, so it'll probably be better to open a separate discussion ticket for it.
